docs/Refactor.md

```diff
@@ -0,0 +1,23 @@
+# Refactor
+
+## Surround with Tags (Wrap)
+
+This refactor command gives the capability to select an XML content and surround it with a given tag. To execute this command you can:
+
+ * use command palette (`Ctrl+P`) and type `Surround`
+
+![Surround with Tags](images/Refactor/SurroundWithTags.gif)
+
+ * use contextual menu
+
+## Surround with Comments
+
+As `Surround with Tags (Wrap)`, you can comments a selected XML content:

```

Comment:
*Similar to* `Surround with Tags (Wrap)`, you can comment~~s~~ *out* the selected XML content:

Reply:
fixed